packages/internals/package.json

```diff
@@ -22,6 +22,8 @@
   "files": [
     "README.md",
     "dist",
+    "!**/libquery_engine*",
+    "!dist/get-generators/engines/*",
```

Comment:
Wouldn't just `!dist/get-generators/engines/*` be enough?

Reply:
On my local computer, I have these, for example 😅 
```
npm notice 33.2MB dist/libquery_engine-darwin-arm64.dylib.node 
npm notice 33.1MB dist/get-generators/engines/1545b76828e10ec7b4630c6ef889b07e389d12b2/libquery_engine-darwin-arm64.dylib.node
npm notice 33.2MB dist/get-generators/engines/39190b250ebc338586e25e6da45e5e783bc8a635/libquery_engine-darwin-arm64.dylib.node                                                      
npm notice 33.2MB dist/get-generators/libquery_engine-darwin-arm64.dylib.node   
```

`get-generators` it's only used for tests and can be completely